<?xml version="1.0"?>
<launch>
    <arg name="vehicle" default="iris_rplidar" />

    <!-- RPLiDAR Setup -->
    <remap from="/rplidar_scan" to="/laser/scan"/>
    <include file="$(find rplidar_python)/launch/rplidar.launch"/>

    <!-- Send the localization data from Optitrack to FCU -->
    <node name="fusion_helper_node" pkg="indoor_navigation" type="fusion_helper.py" output="screen"/>

    <param name="/px4/vehicle_type" type="str" value="$(arg vehicle)"/>
    <rosparam file="$(find indoor_navigation)/config/exp_config.yaml" command="load" ns="/px4"/>
</launch>
